components/lsp/LSP.styled.tsx

```diff
@@ -179,8 +179,8 @@ export const SettleButton = styled(BaseButton)<ILSPButton>`
   font-weight: 600;
   font-family: inherit;
   padding: 0.66rem 0;
-  opacity: ${(props) => (props.showDisabled ? "0.5" : "1")};
-  cursor: ${(props) => (props.showDisabled ? "not-allowed" : "pointer")};
+  opacity: ${(props) => (props.disabled ? "0.5" : "1")};
```

Comment:
nit: you could target the css :disabled selector instead of using props here (which would make the browser handle the transition instead of react)
It would look like this:
`&:disabled {
    opacity: 0.5;
    cursor: not-allowed
  }`

Reply:
ah cool i can give that a try!